### PR TITLE
feat: use rond logger instead of logrus

### PR DESCRIPTION
**Pull request by davidebianchi**

### CODE REVIEW

File: core/input.go [hunk 1]
```diff
@@ -21,9 +21,8 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/rond-authz/rond/logging"
```
Comment (fredmaggiowski) [inline core/input.go:24]: why logging and not logger? Why not an internal package?

File: logging/logrus/logger_test.go [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rondlogrus
+
+import (
+	"testing"
+
+	"github.com/rond-authz/rond/logging"
```
Comment (fredmaggiowski) [inline logging/logrus/logger_test.go:20]:
```suggestion
	"github.com/rond-authz/rond/logging"

```

File: logging/test/logger_test.go [hunk 3]
```diff
@@ -0,0 +1,368 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package test
+
+import (
+	"testing"
+
+	"github.com/rond-authz/rond/logging"
```
Comment (fredmaggiowski) [inline logging/test/logger_test.go:22]:
```suggestion
	"github.com/rond-authz/rond/logging"

```

File: main.go [hunk 4]
```diff
@@ -86,7 +89,7 @@ func entrypoint(shutdown chan os.Signal) {
 		"oasApiPath":  env.TargetServiceOASPath,
 	}).Trace("OAS successfully loaded")
 
-	mongoClient, err := mongoclient.NewMongoClient(env, log)
+	mongoClient, err := mongoclient.NewMongoClient(env, rondlogrus.NewLogger(log))
```
Comment (fredmaggiowski) [inline main.go:92]: Why aren't you submitting the same logger created at line [73](https://github.com/rond-authz/rond/pull/220/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R73)?

File: main_test.go [hunk 5]
```diff
@@ -1799,7 +1799,7 @@ filter_policy {
 			MongoClient: mongoClient,
 		},
 		Registry: registry,
-		Logger:   logrus.NewEntry(logger),
+		Logger:   rondlogrus.NewLogger(logger),
```
Comment (fredmaggiowski) [inline main_test.go:1802]: can't we use the noop logger here?

File: sdk/sdk_test.go [hunk 6]
```diff
@@ -117,6 +102,31 @@ func TestNewFromOas(t *testing.T) {
...
+		require.NoError(t, err)
+		require.NotNil(t, sdk)
+
+		t.Run("and find evaluators", func(t *testing.T) {
+			evaluator, err := sdk.FindEvaluator(logger, http.MethodGet, "/users/")
+			require.NoError(t, err)
+			require.NotNil(t, evaluator)
+		})
+	})
+
```
Comment (fredmaggiowski) [inline sdk/sdk_test.go:129]:
```suggestion
```

useless empty line

File: go.mod [hunk 7]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/gorilla/mux v1.8.0
 	github.com/mia-platform/configlib v1.0.0
-	github.com/mia-platform/glogger/v2 v2.1.3
+	github.com/mia-platform/glogger/v4 v4.0.0
```
Comment (fredmaggiowski) [inline go.mod:11]: 👍

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]

File: main.go (hunk 4 above)
Comment (davidebianchi) [reply main.go:92]: Used the same

File: main_test.go (hunk 5 above)
Comment (davidebianchi) [reply main_test.go:1802]: Yes, we could use but in the main we use logrus so I think it's better for the test

File: core/input.go (hunk 1 above)
Comment (davidebianchi) [reply core/input.go:24]: Yes, but the child package logrus must be used if you use logrus. Also, the test logger could be useful to test, maybe? idk
And the interface must be public

Comment (fredmaggiowski) [reply core/input.go:24]: makes sense, ok

**Review by fredmaggiowski — APPROVED** [review]
LGTM

**Pull request merged**